Dockerfile

```
@@ -7,7 +7,7 @@ ARG TEMPLATE_REPOSITORY=https://github.com/shopware/production
 ENV SHOPWARE_BUILD_DIR /opt/shopware
 
 RUN \
-  if echo "${SHOPWARE_VERSION}" | grep -q '^6.4.*'; then \
+  if echo "${SHOPWARE_VERSION}" | grep -q '6.4.*'; then \
```

I'd prefer if we still keep the caret, as we otherwise might mismatch e.g. the `6.6.4.x` version. But the matching, as currently implemented is wrong :-)

Thank you for investigating.

```suggestion
  if echo "${SHOPWARE_VERSION}" | grep -q '^v6.4.*'; then \
```

---

Hello @aragon999 :)
I updated the PR with your suggestion so now it should behave like the caret operator and major/minor versions should be pinned to version starting with `v6.4` and only the patch is free according to the wildcard(*)